docs/src/fieldexchanger.md

```diff
@@ -0,0 +1,34 @@
+# FieldExchanger
+
+This module contains general functions for the exchange of fields between the atmospheric and surface component models.
+
+The `FieldExchanger` needs to populate the coupler with
+- atmospheric fields (mostly fluxes), via the `import_atmos_fields!` function
+- average surface properties of each coupler gridpoint, via the `import_combined_surface_fields!` function
+
+The component models are updated by broadcasting the coupler fields, via the `update_model_sims!` function. For an update, this function requires that `update_field!` is defined for the particular variable and component model. Currently, we support the:
+- `AtmosModelSimulation`: `albedo`, `surface_temperature`
+    - if calculating fluxes in the atmospheric model: `roughness_momentum`, `roughness_buoyancy`, `beta`
+- `SurfaceModelSimulation`: `air_density`, `turbulent_energy_flux`, `turbulent_moisture_flux`, `radiative_energy_flux`, `liquid_precipitation`, `snow_precipitation`
+
+If an `update_field!` function is not defined for a particular component model, it will be ignored.
+
+
+## FieldExchanger API
+
+```@docs
+    ClimaCoupler.FieldExchanger.import_atmos_fields!
+    ClimaCoupler.FieldExchanger.import_combined_surface_fields!
+    ClimaCoupler.FieldExchanger.update_model_sims!
+    ClimaCoupler.FieldExchanger.update_sim!
+    ClimaCoupler.FieldExchanger.reinit_model_sims!
+    ClimaCoupler.FieldExchanger.step_model_sims!
+```
+
+
+## FieldExchanger Internal Functions
+
+```@docs
+    ClimaCoupler.FieldExchanger.step!
+    ClimaCoupler.FieldExchanger.reinit!
+```
```

Comment:
Good point! But for the release we should decide what we want to do, so as to ensure consistency. I've been using the docs from the other modules as a template. This may not be the best place for it, but I've started compiling a [list of rules](https://github.com/CliMA/ClimaCoupler.jl/wiki/Coupler-Rules) that we may want to enforce in the coupler. Maybe this is something we can discuss / develop further as we go through the clean up tasks for the release 🚀 .